tests/javaRosa.js

```diff
@@ -366,8 +366,8 @@ define([
 
         it("itext widget should not overwrite label with question id", function () {
             util.loadXML("");
-            var q1 = util.addQuestion("Text");
-            util.addQuestion("Text");
+            var q1 = util.addQuestion("Text", "question1");
+            util.addQuestion("Text", "question2");
```

Comment:
Given the video in the PR description, I would expect these IDs to default to the values specified here. Is it now required to always pass a nodeID to `addQuestion()`? If so, what do you think of modifying `util.addQuestion` to get the next default question ID with `form.generate_question_id()` if not specified?